src/services/store.ts

```diff
@@ -28,8 +31,35 @@ export function getFavoriteLabelValuesFromStorage(dsKey: string | unknown, label
   return labelValues;
 }
 
```

Comment:
These are not "store" functions, right? Feel free to create a new service.

Reply:
Created a favorites service